src/stores/appInterface.ts

```diff
@@ -2,19 +2,21 @@ import { useWindowSize } from '@vueuse/core'
 import { defineStore } from 'pinia'
 import { watch } from 'vue'
 
+import { useBlueOsStorage } from '@/composables/settingsSyncer'
+
 const { width: windowWidth, height: windowHeight } = useWindowSize()
 
 export const useAppInterfaceStore = defineStore('responsive', {
   state: () => ({
     width: windowWidth.value,
     height: windowHeight.value,
     configModalVisibility: false,
-    UIGlassEffect: {
+    UIGlassEffect: useBlueOsStorage('UIGlassEffect', {
```

Comment:
This key should be following the same standard we currently have: `cockpit-ui-glass-effect`.